evm/src/NttManager/NttManager.sol

```diff
@@ -135,8 +137,8 @@ contract NttManager is INttManager, NttManagerState {
         if (nativeTokenTransfer.toChain != chainId) {
             revert InvalidTargetChain(nativeTokenTransfer.toChain, chainId);
         }
-
-        TrimmedAmount memory nativeTransferAmount = _nttFixDecimals(nativeTokenTransfer.amount);
+        TrimmedAmount memory nativeTransferAmount =
+            nativeTokenTransfer.amount.untrim(tokenDecimals()).trim(tokenDecimals());
```

Comment:
This should operate from left to right but can we add an extra set of brackets here to be explicit?

Reply:
sure